packages/evolution-backend/src/services/audits/auditChecks/checks/README.md

```
@@ -0,0 +1,166 @@
+
+// VisitedPlace
+VP_M_Geography     // Visited place geography is missing
+
+// Trip
+T_M_Segments       // Trip segments are missing
+
+// Segment
+S_M_Mode           // Segment mode is missing
+```
+
+### Invalid Data Checks (I)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)